test/e2e/specs/realrp/vnetpeering.go

```diff
@@ -78,44 +75,42 @@ var _ = Describe("Peer Vnet tests [Vnet][Real][LongRunning]", func() {
 					},
 				},
 			},
-			Location: &region,
+			Location: &cfg.Region,
 		})
 		Expect(err).ToNot(HaveOccurred())
-		Expect(future).ToNot(BeNil())
```

Comment:
@charlesakalugwu has asked me to check for nils on the return objects.

Reply:
Futures are [structs](https://github.com/Azure/azure-sdk-for-go/blob/07f918ba2d513bbc5b75bc4caac845e10f27449e/services/devtestlabs/mgmt/2016-05-15/dtl/models.go#L8513) and they are always complete when err == nil